bootstrap.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+require_once __DIR__.'/vendor/autoload.php';
+umask(0002);
+$defaultFile = __DIR__.'/config/config.php';
+$legacyFile = __DIR__.'/config/'.getenv('ENVIRONMENT_NAME').'.php';
+if (file_exists($defaultFile)) {
+    $config = include $defaultFile;
```

Comment:
Take a look at https://github.com/elifesciences/recommendations/pull/100 where I'm working on a slightly different approach.

Reply:
Should be easy to add the environment-specific files, merge them, move the main one up a folder